tests/countries/test_thailand.py

```diff
@@ -559,33 +588,79 @@ def test_khao_phansa(self):
         )
 
     def test_raeknakhwan(self):
+        # Boiler Plates
```

Comment:
With #1347, there will be no need for this, only actual :)

Reply:
Fixed now with #1347 changes.